account/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.urls import include, path
+
+from . import views
+
+app_name = 'account'
+
+register_patterns = ([
```

Comment:
二つに分けている理由はありますか

Reply:
この部分をコメントアウトすると以下のエラーメッセージが出力されてしまうため必要だと考えています．
![image](https://user-images.githubusercontent.com/47509184/150484405-e22e4b29-c655-4fde-ba18-20eae637cda4.png)

Reply:
okです。下記でも動作すると思いますが、お任せします。
```suggestion
urlpatterns = [
    path('', views.start_view, name='start'),
    path('register/', views.register_view, name='register'),
    path('register/complete/', views.complete_view, name='complete')
    path('login/', views.login_view, name='login'),
]
```

Reply:
上記のコードで動作を確認しました．
修正しておきます．